subiquity/ui/views/filesystem/guided.py

```diff
@@ -155,12 +155,12 @@ class TPMChoice:
             enabled=False, default=False,
             help=_("The model being installed prefers but does not require "
                    "TPM backed full-disk encryption and it is not available "
-                   "on this device")),
+                   "on this device (the reason given was \"{reason}\")")),
         StorageSafety.PREFER_UNENCRYPTED: TPMChoice(
             enabled=False, default=False,
             help=_("The model being installed does not prefer TPM backed "
                    "full-disk encryption and it is not available on this "
-                   "device.")),
+                   "device (the reason given was \"{reason}\").")),
```

Comment:
Minor:
As a non-native speaker, I find it confusing whether the reason given applies to:
* FDE not being available on this device
* The model _not_ preferring TPM backed FDE

but an example would probably make it clear, so it's just a potential obstacle for translators I guess :)

Reply:
The messages are not especially likely to be human friendly (which is why they were not shown originally) but here is the example from the sample data:

![image](https://user-images.githubusercontent.com/672022/211224595-1a5996a1-692b-450d-8622-91d06715ea1b.png)

In this case at least, I think it's reasonably clear that this is a reason for FDE not being available. I can add a note for the translator.